docs/program-setup/serial/serial_communication.md

```diff
@@ -6,57 +6,60 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
 
 This allows you to:
 
-- Print out messages to a [host PC terminal (useful for debugging)](#terminal-applications).
+- Print out messages to a host PC terminal (useful for debugging).
 - Read input from the host PC keyboard.
 - Communicate with applications and programming languages running on the host PC that can communicate with a serial port. Examples are Perl, Python and Java.
 
-## Hello, world
+## Hello World - printing messages
 
-This program prints a "Hello World" message that you can view on a [terminal application](#using-terminal-applications). Communication over the USB serial port uses the standard serial interface. Specify the internal (USBTX, USBRX) pins to connect to the serial port routed over USB:
+This program prints a "Hello World!" message that you can view on a serial terminal. Mbed OS redirects any `printf()` statements to the board's debug USB serial.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_HelloWorld/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_HelloWorld/main.cpp)
-
-## Using terminal applications
-
-Terminal applications run on your host PC. They provide a window where your Mbed board can print and where you can type characters back to your board.
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/main.cpp)
 
-<span class="tips">**Serial configuration:** The standard setup for the USB serial port is 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1)</span>
+### Viewing outputs with `mbed sterm`
 
-### Installing an application for Windows
+Serial terminals run on your host PC. They provide an interface where your Mbed board can print and where you can type characters back to your board.
 
-There are many terminal applications for Windows, including:
+Mbed CLI has built-in serial support. To open a serial terminal when _one_ Mbed board is connected and using the default baud rate (9600):
```

Comment:
```suggestion
Mbed CLI provides a serial terminal which is configured with a default baud rate of `9600`. When single board is connected, simply run: 
```

Why say that it works only when one board is connected?

Reply:
As a step-by-step guide, we want to show the simplest scenario first and introduce more options later